imap_processing/ultra/utils/ultra_l1_utils.py

```diff
@@ -36,6 +36,7 @@ def create_dataset(
         coords = {
             "spin_number": data_dict["spin_number"],
             "energy_bin_geometric_mean": data_dict["energy_bin_geometric_mean"],
+            "epoch": data_dict["spin_start_time"],
```

Comment:
Does this need to be the mean time, rather than the start?

Reply:
Perfect, well explained. Maybe add a comment noting why it must be start time to align with the universal spin table.